### PR TITLE
SSE support for getObject

### PR DESCRIPTION
**Pull request by JohannGit**
Currently minio-js has SSE support for putObject but not for getObject.
I had to do a npx patch for my project to support it, but I'll be glad if it's part of the minio-js directly.

This PR is related to very old issues but SSE support for getObject never reached master branch: 

- https://github.com/minio/minio-js/issues/1019
- https://github.com/minio/minio-js/pull/902
- https://github.com/minio/minio-js/issues/851
- https://github.com/minio/minio-js/issues/726
- https://github.com/minio/minio-js/issues/723

This PR brings the ability to give SSE headers to the getObject call via the metaData parameter, the same way putObject does.

### CODE REVIEW

File: src/internal/client.ts [hunk 1]
```diff
@@ -953,14 +953,15 @@ export class TypedClient {
     bucketName: string,
     objectName: string,
     getOpts: VersionIdentificator = {},
+    metaData: ItemBucketMetadata = {},
```
Comment (prakashsvmx) [inline src/internal/client.ts:956]: the option for SSE should be part of getOpts. you can update the type and add the SSE fields.

**Review by prakashsvmx — CHANGES_REQUESTED** [review]
Thank you for the contribution @JohannGit . Could you check the comment ?

File: src/internal/client.ts (hunk 1 above)
Comment (JohannGit) [reply src/internal/client.ts]: Hi @prakashsvmx , I've made the changes to include SSE headers in getOpts. Tell me if it's better

**Comment by JohannGit** [comment]
> Thank you for the contribution @JohannGit . Could you check the comment ?

Hi @prakashsvmx , I've made the changes to include SSE headers in getOpts. Tell me if it's better

File: src/internal/client.ts (hunk 1 above)
Comment (prakashsvmx) [reply src/internal/client.ts]: Yes. Please resolve the conflicts. :+1:

Comment (JohannGit) [reply src/internal/client.ts]: Here it is @prakashsvmx. Conflicts resolved 👍

**Comment by JohannGit** [comment]
@prakashsvmx sorry CI were not successful cause I forgot to use prettier.
Now it should be all good 👍

**Review by prakashsvmx — APPROVED** [review]

**Review by kaankabalak — APPROVED** [review]

**Pull request merged**